NAMESPACE

```diff
@@ -10,5 +10,4 @@ export(scaffoldWidget)
 export(shinyRenderWidget)
 export(shinyWidgetOutput)
 export(sizingPolicy)
-import(RJSONIO)
 import(htmltools)
```

Comment:
We should not need this in the NAMESPACE, since all `jsonlite` functions are being used as `jsonlite::toJSON` etc.